onnxruntime/test/shared_lib/utils.h

```diff
@@ -156,3 +166,58 @@ struct MyCustomOpWithAttributes : Ort::CustomOpBase<MyCustomOpWithAttributes, My
  private:
   const char* provider_;
 };
+
+//Slice array of floats or doubles between [from, to) and save to output
+struct SliceCustomOpKernel {
```

Comment:
Moved this for test_inference.cc

Reply:
May be better to rename files to custom_op_utils.h/.cc given their current content

---
In reply to: [591514397](https://github.com/microsoft/onnxruntime/pull/6971#discussion_r591514397) [](ancestors = 591514397)

Reply:
done